src/PatientChecking/PatientChecking/Views/Patient/Index.cshtml

```diff
@@ -33,39 +37,53 @@
                 <div class="col l-4">
                     <div class="item patient-item1">
                         <div>
-                            <img src="~/Image/no-avatar.jpg" alt="Avatar" class="patient-img" />
+                            @if (string.IsNullOrEmpty(item.AvatarLink))
+                            {
+                                <img src="~/Image/no-avatar.jpg" alt="Avatar" class="patient-img" />
+                            }
+                            else
+                            {
+                                <img src="@item.AvatarLink" alt="Avatar" class="patient-img" />
```

Comment:
It can be: `<img src="@(string.IsNullOrEmpty(item.AvatarLink) ? "~/Image/no-avatar.jpg" : @item.AvatarLink)" alt="Avatar" class="patient-img" />`

Reply:
This section has been changed. Please review this @vietvo361